docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -42,6 +42,21 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 
 ### Parameters
 
+* **grouping_expression**
+
+    Specifies the criteria based on which the rows are grouped together. The grouping of rows is performed based on
+    result values of the grouping expressions. A grouping expression may be a column alias, a column position
+    or an expression.
+    
+* **group_expression_with_grouping_analytics**    
```

Comment:
We don't need this if we do https://github.com/apache/spark/pull/30212/files#r601146756

Reply:
Done